packages/language-server/src/importPackage.ts

```diff
@@ -48,3 +49,14 @@ export function importSvelteIntegration(fromPath: string): typeof svelte | undef
 export function importVueIntegration(fromPath: string): typeof vue | undefined {
 	return importEditorIntegration('@astrojs/vue', fromPath);
 }
+
+export function importPrettier(fromPath: string): typeof prettier {
+	// This shouldn't ever fail, because we bundle Prettier in the extension itself
+	const prettierPkg = getPackagePath('prettier', [fromPath, __dirname])!;
+
+	return require(prettierPkg);
+}
+
+export function importPrettierPlugin(fromPath: string): string {
```

Comment:
More of a function naming thing, but I find it a little unexpected that importPrettier imports it into the module graph but this one does not.

Reply:
This isn't blocking to me, just an observation.